src/index.ts

```diff
@@ -83,7 +83,7 @@ const withDefaults: (
     steps: draft.steps ?? 50,
     cfgScale: draft.cfgScale ?? 7,
     samples: draft.samples ?? 1,
-    outDir: draft.outDir ?? path.join(process.cwd(), '.out', requestId),
+    outDir: draft.outDir ?? path.join('.out', requestId),
```

Comment:
see #3, this change would technically be breaking

also, why?

Reply:
Same, I was trying to deconstruct the filePath so I can move the files and rebuild a src for the img tag. Same noobish reason for the previous requestId request. 

But actually I think I can pass the custom build directory through the API using the API variables like **davmixcool** suggested...
  const { result,images }:any = await generateAsync({
       prompt: prompt,
       apiKey: apikey,
       outDir: 'public/images',
       debug: debug,
       samples: samples,
       engine: engine,
       seed: seed,
       width: width,
       height: height,
       diffusion: diffusion,
       steps: steps,
       cfgScale: cgf_scale,
 })

I'm going to try this.